### PR TITLE
tazeek/add_reason_sanctions

### PR DESCRIPTION
**Pull request by tazeek**

### CODE REVIEW

File: lib/Data/Validate/Sanctions.pm [hunk 1]
```diff
@@ -28,8 +31,29 @@ sub new {    ## no critic (RequireArgUnpacking)
...
+
+    $self->_save_data if $updated;
+    return;
+}
+
+sub last_updated {
+    my $self = shift;
+    my $list = shift;
+    return $list ? $self->{_data}->{$list}->{updated} : $self->{last_time};
+}
```
Comment (zakame) [inline lib/Data/Validate/Sanctions.pm]: Why were these moved up?  Seems unneeded change here...

Comment (tazeek) [reply lib/Data/Validate/Sanctions.pm]: In one of my old PRs (not sure which), @code4pay mentioned that private subs should be at the start

Comment (tazeek) [reply lib/Data/Validate/Sanctions.pm]: Oh wait -.- my bad. they should be at the bottom, as per: https://github.com/binary-com/perl-Data-Validate-Sanctions/pull/23#discussion_r226208767 Making changes now
